File: back-end/src/FiapMC.Business/Models/Restaurante.cs
using System.Collections.Generic;

namespace FiapMC.Business.Models
{
    public class Restaurante : Entity
    {
        public string Nome { get; set; }
        public string Documento { get; set; }
        public TipoRestaurante TipoRestaurante { get; set; }
        public Endereco Endereco { get; set; }
        public bool Ativo { get; set; }

        /* EF Relations */
        public IEnumerable<Receita> Receitas { get; set; }
    }
}

File: front-end/src/app/restaurante/models/restaurante.ts
import { Endereco } from './endereco';
import { Receita } from 'src/app/receita/models/receita';

export class Restaurante {
    id: string;
    nome: string;
    documento: string;
    ativo: boolean;
    tipoRestaurante: number;
    endereco: Endereco;
    receitas: Receita[]
}
